setup.py

```diff
@@ -18,7 +18,7 @@
     packages = ['jupyter_sphinx'],
     install_requires = [
         'Sphinx>=0.6',
-        'ipywidgets>=6.0.0',
+        'ipywidgets>=7.0.0',
         'IPython',
         'nbconvert>=5.4',
```

Comment:
`nbconvert>=5.5`

Reply:
Actually not sure about this one: embedding widgets will only work on `5.5`, however we might want to be lenient and allow for older systems that don't need to use widgets.

Reply:
Shouldn't you do this now that you pushed b507db458dff2686eebd087e7e7a804fab70cad2?

Reply:
Another option would be to introduce an `extras_require`, and make `nbconvert>=5.5` as well as `ipywidgets>=7.0.0` a prerequisite for a `widgets` extra.

The motivation for this would be a smaller dependency footprint

Reply:
In other words: everything will still work on `nbconvert>=5.4`, except widgets won't be embedded.

Reply:
I believe it is fine requiring `nbconvert 5.5`. The thing about allowing two versions with notably different behaviors is that it may break expectations. So for the principle of least surprise, I prefer >=5.5.

Reply:
Done